package.json

```diff
@@ -18,6 +18,8 @@
   "scripts": {
     "docs:build": "node build/documentation/copy-package-readme.js",
     "docs:dry-run": "node build/documentation/copy-package-readme.js --dry-run",
+    "docs:publish": "node build/documentation/generate-typedocs.js",
```

Comment:
"build" generates readme's and "publish" generates typedocs.  I think what i'd like to see is "docs:build" that compiled all of the documentation (readmes, apis, etc).  Each of the parts also have scripts of their own when wanting to just test readmes or api for example. Publish we should add later for the actual uploading to gh-pages.

Reply:
So in that case I should maybe do something like:

`"docs:build": build/documentation/generate-typedocs.js` -> runs `generate-typedocs` script (which generates all docs)
`"docs:build:readme": node build/documentation/copy-package-readme.js`-> only build readme files with `copy-package-readme` script
`"docs:build:typedoc": node build/documentation/generate-typedocs.js` -> meant to only build api docs, however `generate-typedocs` currently builds all documentation (it rebuilds the existing `doc/en/package` readme files to prevent appending multiple versions of the api doc link them). So this one might be misleading unless the readme and api doc process were better separated.

I have an new idea for how we could separate the typedoc and readme process while avoiding this appending issue, gonna message you in the discord about it though.

Reply:
Just in case we swap out typedoc one day I may call it "docs:build:api" to limit impact if we change tech in the future which then would require CI/CD changes.

At might be good to separate out typedocs vs readmes. @nicholasrice thoughts on what he thinks.